internal/resources/resource_host.go

```diff
@@ -44,6 +44,7 @@ const (
 	hPortalCommOkay       = "portal_comm_okay"
 	hPwrState             = "power_state"
 	hLabels               = "labels"
+	hDiscoveryIPs         = "discovery_ips"
```

Comment:
since in the API, it is the inner field of 'iscsi', rename the label to reflect that iscsi_discovery_ip?